modules/backend/widgets/MediaManager.php

```diff
@@ -926,13 +927,15 @@ protected function getCurrentFolder(): string
      */
     protected function setFilter(string $filter): void
     {
-        if (!in_array($filter, [
+        if (
+            !in_array($filter, [
             self::FILTER_ALL,
             MediaLibraryItem::FILE_TYPE_IMAGE,
             MediaLibraryItem::FILE_TYPE_AUDIO,
             MediaLibraryItem::FILE_TYPE_DOCUMENT,
-            MediaLibraryItem::FILE_TYPE_VIDEO
-        ])) {
+            MediaLibraryItem::FILE_TYPE_VIDEO,
```

Comment:
Should this be indented one more level?

Reply:
I believe that's another rule, but not one I'm using right now. Will take a look.

Reply:
Added rule.